.env.example

```
@@ -0,0 +1,6 @@
+# 環境変数サンプルファイル
+
+ENV= # 環境(ci/development/production)
```

dotenv-linter (3.3.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)